src/adzerk/boot_cljs.clj

```diff
@@ -45,14 +45,8 @@
         out-dir       (if-not keep-out?
                         (core/temp-dir!)
                         (apply io/file tmp-dir (remove empty? [js-parent output-dir])))
-        base-opts     {:libs          []
-                       :externs       []
-                       :preamble      []
-                       :foreign-libs  []
-                       :warnings      (not no-warnings)
-                       :output-to     (.getPath js-out)
+        required-opts {:output-to     (.getPath js-out)
```

Comment:
The `:libs`, `:externs`, `:preamble` & `:foreign-libs` keys are all set to their default values so there's no reason to set them at all right?